python/k4FWCore/ApplicationMgr.py

```diff
@@ -45,18 +51,22 @@ def __init__(self, **kwargs):
                 continue
             props = conf.getPropertiesWithDescription()
             reader = writer = None
-            add_reader = add_writer = False
+            add_reader = False
             for alg in self._mgr.TopAlg:
                 if isinstance(alg, Reader):
                     reader = alg
                 elif isinstance(alg, Writer):
                     writer = alg
-            if reader is None and props["input"][0]:
+            if reader is None and (props["input"][0] or props["Input"][0]):
```

Comment:
Add a comment that 'input' is deprecated and can be removed soon?

Reply:
Comment added